cli/README.md

```diff
@@ -3,9 +3,10 @@
 This utility manages Solana Foundation Delegation Program registrations.
 
 ## Installation
-Install Rust from https://rustup.rs/, then run
 ```bash
-$ sudo apt install libudev-dev
+$ su sol
+$ cd
+$ sudo apt install libudev-dev libssl-dev cargo pkg-config 
```

Comment:
cargo should not be installed this way.  The distro cargo versions are always very old.  Rustup.rs is the way